stdlib/config-api/src/main/ballerina/src/config/config.bal

```diff
@@ -149,6 +149,20 @@ public function getAsMap(@untainted string key) returns map<any> {
     }
 }
 
+# Retrieves the specified configuration value as an array. If there is no mapping, an empty array will be returned.
+#
+# + key - The configuration to be retrieved
+# + return - Configuration value mapped by the key
+public function getAsArray(@untainted string key) returns any[] {
+    var value = get(key, ARRAY);
+    if (value is any[]) {
+        return value;
+    } else {
+        error err = error("Invalid value. Expected an 'any[]'.");
```

Comment:
Is there any way to improve this message like 'Expect 1111 got 2222' format?

Reply:
Removed this code block.